src/ext/graphql/transmodelapi/.graphqlconfig

```diff
@@ -1,17 +0,0 @@
-{
-  "name": "Entur Transmodel API GraphQL Schema",
```

Comment:
I'm not sure what this file was used for but it was confusing my Intelliq enormously when writing queries.